src/main/java/appeng/core/worlddata/DimensionData.java

```diff
@@ -153,19 +153,20 @@ public void setStoredSize( final int dim, final int targetX, final int targetY,
 	@Override
 	public void sendToPlayer( @Nullable final NetworkManager manager )
 	{
-		if( manager != null )
+		// if( manager != null )
+		// {
+		// for( final int newDim : this.config.get( PACKAGE_DEST_CATEGORY, PACKAGE_KEY_CATEGORY, PACKAGE_DEF_CATEGORY
+		// ).getIntList() )
+		// {
+		// manager.scheduleOutboundPacket( ( new PacketNewStorageDimension( newDim ) ) );
+		// }
+		// }
+		// else
+		// {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)